packages/engine-multi/src/api/call-worker.ts

```diff
@@ -60,7 +60,14 @@ export default function initWorkers(
   };
 
   // This will force termination instantly
-  api.closeWorkers = () => workers.terminate(true);
+  api.closeWorkers = () => {
+    workers.terminate(true);
+
+    // Defer the return to allow workerpool to close down
+    return new Promise((done) => {
+      setTimeout(done, 100);
```

Comment:
This seems to stabilise the tests, but boy it's a bit of a sledgehammer!

With it at 100, test have passed 5 times in a row

![image](https://github.com/OpenFn/kit/assets/7052509/91418faa-1023-4412-8b8f-297dc1bab05d)